worker/uniter/relation/resolver_test.go

```diff
@@ -1698,11 +1722,6 @@ func (s *mockRelationResolverSuite) expectState(st relation.State) {
 	exp.State(st.RelationId).Return(&st, nil)
 }
 
-func (s *mockRelationResolverSuite) expectIsPeerRelationFalse(id int) {
```

Comment:
I'd advocate removing all of these helpers. They are too shallow to pay their way, and are inflexible into the bargain.